.buildkite/scripts/steps/artifacts/docker_image.sh

```diff
@@ -29,6 +29,29 @@ node scripts/build \
   --skip-docker-ubi \
   --skip-docker-cloud \
   --skip-docker-contexts
+
+echo "--- Tag images"
+docker rmi "$KIBANA_IMAGE"
+docker load < "target/kibana-$BASE_VERSION-docker-image.tar.gz"
+docker tag "$KIBANA_IMAGE" "$KIBANA_IMAGE-amd64"
+
+docker rmi "$KIBANA_IMAGE"
+docker load < "target/kibana-$BASE_VERSION-docker-image-aarch64.tar.gz"
+docker tag "$KIBANA_IMAGE" "$KIBANA_IMAGE-arm64"
+
+echo "--- Push images"
+docker image push "$KIBANA_IMAGE-arm64"
+docker image push "$KIBANA_IMAGE-amd64"
```

Comment:
I believe the elasticsearch team is using `-x86_64` and `-aarch64`. After confirming this is the case, should we match their behaviour ? \cc @mgreau

Reply:
> @mistic it's following the naming convention of our release related docker images - https://www.docker.elastic.co/r/kibana

+1 to keep using the same naming convention. 
That should be transparent for end-users as everyone should reference the main tag and let the container runtime pulls the right tag based on the underlying architecture.